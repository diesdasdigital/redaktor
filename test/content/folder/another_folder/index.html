{
  "markdown": "",
  "path": "folder/another_folder/index",
  "updatedAt": "2020-08-28T12:38:38.009Z",
  "createdAt": "2020-08-28T12:38:38.009Z"
}