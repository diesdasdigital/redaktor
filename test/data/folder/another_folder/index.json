{
  "default": {
    "required": {
      "title": "Another Folder",
      "pageType": "code-preview"
    },
    "fields": {},
    "components": []
  }
}
