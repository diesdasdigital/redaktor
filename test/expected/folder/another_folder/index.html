<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Another Folder</title>
  <meta name="viewport" content="width=device-width,initial-scale=1" />
</head>
<body>
  <pre>{
  "data": {
    "default": {
      "required": {
        "title": "Another Folder",
        "pageType": "code-preview"
      },
      "fields": {},
      "components": []
    }
  },
  "path": "folder/another_folder/index"
}</pre>
</body>
</html>